### DISCUSSION TITLE
Detect Quit Button

### DISCUSSION THREAD

**Discussion by Ramatonic** [SafeExamBrowser/seb-win-refactoring#413] [Q&A] [answered]
Hi,
I set quitURL in SEB configuration and I point to it with a button in my exam page; in this way I can log when a student exit the exam.
I'd like to know if there is a way to call the same URL when the student use the SEB Quit Button.
Obviously he had to know the quit password (that we set) to use it, but we'd like to log when he use it to exit exam anyway.

Thanks in advance

**Comment by dbuechel** [comment, marked as answer]
No, there is currently no such functionality, the only way would be to collect that information from the log files on the client machines. But you could monitor the termination of SEB clients using SEB-Server.

**Reply by Ramatonic** [reply]
Thanks for the answer.
